packages/nx/src/command-line/add/command-object.ts

```diff
@@ -0,0 +1,28 @@
+import { CommandModule } from 'yargs';
+
+export interface AddOptions {
+  packageSpecifier: string;
+}
+
+export const yargsAddCommand: CommandModule<
+  Record<string, unknown>,
+  AddOptions
+> = {
+  command: 'add <packageSpecifier>',
+  describe: 'Install a plugin and initialize it.',
+  builder: (yargs) =>
+    yargs
+      .positional('packageSpecifier', {
+        type: 'string',
+        description: 'The name of an installed plugin to query',
```

Comment:
I don't have a perfect suggested replacement here but this description doesn't feel all that clear?

The plugin isn't necessarily installed? And it doesn't mention that the packageSpecifier can contain a version.

Maybe something like:

```
The name (and optional version) of the plugin to install and initialize
```

Reply:
🤦🏻 copy and paste mistake. I forgot to update the text.